target/linux/ramips/dts/ATEL-U270.dts

```diff
@@ -0,0 +1,177 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+/ {
+	compatible = "ATEL-U270", "ralink,mt7620a-soc";
+	model = "ATEL U270";
+
+	gpio-leds {
+		compatible = "gpio-leds";
+		wan {
+			label = "atel-u270:orange:wan";
+			gpios = <&gpio2 27 1>;
+		};
+		wifi {
+			label = "atel-u270:orange:wifi";
+			gpios = <&gpio3 0 1>;
+		};
+		wps {
+			label = "atel-u270:orange:wps";
+			gpios = <&gpio2 28 1>;
+		};
+		power {
+			label = "atel-u270:orange:power";
+			gpios = <&gpio2 20 1>;
+		};
+		power-red {
+			label = "atel-u270:red:power";
+			gpios = <&gpio2 21 1>;
+		};
+  signal1 {
+   label = "atel-u270:white:signal1";
+   gpios = <&gpio2 26 1>;
+  };
+  signal2 {
+   label = "atel-u270:white:signal2";
+   gpios = <&gpio2 23 1>;
+  };
+  signal3 {
+   label = "atel-u270:white:signal3";
+   gpios = <&gpio2 24 1>;
+  };
+  signal4 {
+   label = "atel-u270:white:signal4";
+   gpios = <&gpio2 25 1>;
+  };
+  signal5 {
+   label = "atel-u270:white:signal5";
+   gpios = <&gpio2 22 1>;
+  };
+  signal-blue {
+   label = "atel-u270:blue:signal";
+   gpios = <&gpio1 7 1>;
+  };
+  signal-red {
+   label = "atel-u270:red:signal";
+   gpios = <&gpio1 8 1>;
+  };
+  signal-green {
+   label = "atel-u270:green:signal";
+   gpios = <&gpio1 9 1>;
+  };
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+/*		power {
+			label = "power";
+			gpios = <&gpio2 31 1>;
+			linux,code = <0x198>;
+		};
+		wps {
+			label = "wps";
+			gpios = <&gpio2 30 1>;
+			linux,code = <0x211>;
+	 };
+  */
+	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+/*		usbpower {
+			gpio-export,name = "usbpower";
+			gpio-export,output = <1>;
+			gpios = <&gpio2 12 1>;
+		};*/
+	};
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&gpio2 {
+	status = "okay";
+};
+
+&gpio3 {
+	status = "okay";
+};
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		linux,modalias = "m25p80", "w25q256";
+		spi-max-frequency = <10000000>;
+
+		partition@0 {
+			label = "u-boot";
+			reg = <0x0 0x30000>;
+			read-only;
+		};
+
+		partition@30000 {
+			label = "u-boot-env";
+			reg = <0x30000 0x10000>;
+			read-only;
+		};
+
+		factory: partition@40000 {
+			label = "factory";
+			reg = <0x40000 0x10000>;
+			read-only;
+		};
+
+		partition@50000 {
+			label = "firmware";
+			reg = <0x50000 0x7b0000>;
+		};
+	};
+};
+
+&pinctrl {
+	state_default: pinctrl0 {
+		default {
+			ralink,group = "i2c", "uartf", "rgmii1", "rgmii2", "ephy", "wled", "nd_sd";
```

Comment:
Please drop `i2c`, `uartf` and `ephy`. The pins of these groups are not used as GPIOs in your dts.